modules/sandboxed-containers-viewing-workloads-from-cli.adoc

```diff
@@ -4,4 +4,17 @@
 
 [id="sandboxed-containers-viewing-workloads-from-cli_{context}"]
 
-= Viewing sandboxed containers workloads from the CLI
+= Viewing {sandboxed-containers-first} workloads from the CLI
+
+You can view the `runtimeClass` that the pods for your workloads use from the CLI.
+
+.Prerequisites
+
+* You have {product-title} 4.8 installed on your cluster.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Inspect the `runtimeClassName` field on the pod to see a pod running on {sandboxed-containers-first} versus a normal container.
```

Comment:
For a one-step procedures, use `*` rather than `.`. You could also likely combine the substep into the step (on a new line with `+` to connect it), since it's more of a result than a step.